cfme/utils/appliance/__init__.py

```diff
@@ -2443,6 +2443,18 @@ def provider_based_collection(self, provider, coll_type='vms'):
             raise ValueError('No support for coll_type: "{}" collection name lookup'
                              .format(coll_type))
 
+    def _switch_migration_ui(self, enable):
+        self.update_advanced_settings({'product': {'transformation': enable}})
```

Comment:
**REQUIRED:** I think the better action here is to explicitly log out, before restarting the evm service.